.idea/cssdialects.xml

```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I believe even if this change is needed, it is out of scope of this pull request; May we revert this not related change?

Reply:
Yeah, sure, I actually don't know how that happened

Reply:
fixed